### PR TITLE
Feature/credit card payments

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

## Features
- Addition of the `credit_card_payment_txn` (enabled/disabled using the `using_credit_card_payment_txn` variable) source as well as the accompanying staging models. This source includes all credit card payment transactions and will be used in downstream General Ledger generation to ensure accurate reporting of all transaction types.
  >**Note**: the `credit_card_payment_txn` source and models are disabled by default. In order to enabled them, you will want to set the `using_credit_card_payment_txn` variable to `true` in your dbt_project.yml.

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide an explanation as to how the change is non-breaking below.)

This PR will only add a new intermediate and component in the GL model that is disabled by default. This will not cause existing runs to fail.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature #47 
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
💳 
**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-jamie — APPROVED** [review]
looks good! left a couple of docs-related comments

just flagging that this package still uses the ~= syntax in the requirements.txt file if we want to change it to >=

note: i do not have access to our quickbooks schema, so i did not run this myself as the customer is also testing it out and circle ci has passed

File: models/stg_quickbooks__credit_card_payment_txn.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
+--To disable this model, set the using_invoice_bundle variable within your dbt_project.yml file to False.
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply models/stg_quickbooks__credit_card_payment_txn.sql:1]: Good catch, it should be!

File: models/tmp/stg_quickbooks__credit_card_payment_txn_tmp.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+--To disable this model, set the using_credit_card_payment_txn variable within your dbt_project.yml file to False.
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply models/tmp/stg_quickbooks__credit_card_payment_txn_tmp.sql:1]: It should indeed, swapping this over. Thanks for the catch.

**Comment by fivetran-joemarkiewicz** [comment]
Thanks for reviewing @fivetran-jamie! I just updated the package based off your notes and will plan to merge with the dbt-utils and dbt-core updates.

**Pull request merged**